static/app/components/replays/replayProcessingErrors.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled';
+
+import {Alert} from 'sentry/components/alert';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+interface Props {
+  className?: string;
+}
+
+export default function ReplayNotFound({className}: Props) {
+  return (
+    <StyledAlert type="error" showIcon className={className}>
+      <Heading>{t('Replay Not Found')}</Heading>
+      <p>{t('The replay you are looking for was not found.')}</p>
+      <p>{t('The replay might be missing events or metadata.')}</p>
+      <p>
+        {t(
+          'Or there may be an issue loading the actions from the server, click to try loading the Replay again.'
+        )}
+      </p>
+      <ul>
+        <li>
+          {t(
+            `If you followed a link here, try hitting back and reloading the
+           page. It's possible the resource was moved out from under you.`
+          )}
+        </li>
+        <li>
+          {tct('If all else fails, [link:contact us] with more details', {
```

Comment:
Should this pop open the feedback widget? :)

Reply:
It wouldn't work for SaaS

Reply:
i can, combined with https://github.com/getsentry/sentry/pull/61411, move the feedback button stuff into getsentry and use a sentry-getsentry hook to insert a feedback link here.

i'd followup with all that though.

Reply:
it might be simpler to log an error when it happens, so we can checkout the urls/replays and investigate for ourselves.

Reply:
Yeah error would be best

Reply:
Updated to capture errors, check it out and lmk